README.md

```diff
@@ -17,190 +17,88 @@ By [Fauna, Inc](http://faunadb.com).
 
 ## Inroduction
 
-This repository contains FaunaDB driver for Swift language. It provides high level abstractions that allows us to work with fauna DB efficiently and without the need to deal with rest messages, networking errors, data encoding, decoding and so on.
-
-> working on another language? Take a look at our github account, we provide Fauna DB drivers for many popular languages and many others are coming. Don't hesitate to contact us if the language you are working on is not supported yet. Notice you can always use the [REST API](https://faunadb.com/documentation/rest) directly.
+This repository contains FaunaDB driver for Swift language. It provides high level abstractions that allows us to work with Fauna DB efficiently and without the need to deal with rest messages, networking errors, data encoding, decoding and so on.
 
 ## Requirements
 
-* iOS 8.0+
+* iOS 8.0+ | OSX 10.9+ | tvOS 9.2+ | watchOS 2.2+
 * Xcode 7.3.1+
 * Swift 2.2
 
 ## Getting started
 
-1. Install the driver first. We can install the driver either by using [Carthage](https://github.com/Carthage/Carthage), [CocoaPods](https://cocoapods.org/) or manually as Embedded Framework. Go to [Installation](#installation) section to know how.
+1. If you haven’t already, [sign up for an account](https://faunadb.com/#signup-modal) to start using FaunaDB.
 
-2. If you haven’t already, [sign up for an account](https://faunadb.com/#signup-modal) to start using FaunaDB.
+2. Driver installation. We can install the driver either by using [Carthage](https://github.com/Carthage/Carthage), [CocoaPods](https://cocoapods.org/) or manually as Embedded Framework. Go to [Installation](#installation) section to know how.
 
 3. From here, Fauna DB [tutorial](https://faunadb.com/tutorials) will walk you through step by step how to get started with Fauna DB.
```

Comment:
How about:

FaunaDB [tutorials](https://faunadb.com/tutorials) provide a task-oriented introduction to the features of FaunaDB to get you up and running quickly.